### PR TITLE
Add dynamic status dot and enhance ChatGPT payload configuration

### PR DESCRIPTION
**Pull request by ItsSimko**
## Summary by Sourcery

Add a dynamic status dot feature to visually represent Docker container statuses and enhance the ChatGPT payload configuration with additional parameters for improved customization.

New Features:
- Introduce a dynamic status dot for Docker containers, updating their visual status based on their current state.

Enhancements:
- Enhance ChatGPT payload configuration by adding parameters like temperature, top_p, top_k, best_of, and tfs, allowing for more customizable AI interactions.

### CODE REVIEW

File: src/FreeScribe.client/UI/MainWindowUI.py [hunk 1]
```diff
@@ -56,6 +56,11 @@ def create_docker_status_bar(self):
 
         # Add status dot for LLM (default: red)
         llm_dot = tk.Label(self.docker_status_bar, text='●', fg='red')
+        if self.logic.container_manager.client is not None:
```
[the remark this answers is not in the document]
Comment (ItsSimko) [reply src/FreeScribe.client/UI/MainWindowUI.py]: https://github.com/ClinicianFOCUS/FreeScribe/pull/33/commits/9b464a3f0d931b770283686d268e36e8c5fb6d48

**Comment by ItsSimko** [comment]
@sourcery-ai review

File: src/FreeScribe.client/client.py [hunk 2]
```diff
@@ -466,6 +466,11 @@ def send_text_to_chatgpt(edited_text):
         "messages": [
             {"role": "user", "content": edited_text}
         ],
+        "temperature": float(app_settings.editable_settings["temperature"]),
```
[the remark this answers is not in the document]
Comment (ItsSimko) [reply src/FreeScribe.client/client.py]: https://github.com/ClinicianFOCUS/FreeScribe/pull/33/commits/44e2aa7307f394fdd7ce0b5ce8443fe266aa171f

Very basic error handling that needs to be revisited. Issue ticket created.

File: src/FreeScribe.client/UI/MainWindowUI.py [hunk 3]
```diff
@@ -190,7 +195,7 @@ def _show_welcome_message(self):
         self._show_md_content(self._get_file_path('welcome.md'), 'Welcome', True)
 
     def _get_file_path(self, *file_names):
-        return os.path.join('src', 'FreeScribe.client', 'markdown', *file_names)
+        return os.path.join('markdown', *file_names)
```
[the remark this answers is not in the document]
Comment (ItsSimko) [reply src/FreeScribe.client/UI/MainWindowUI.py:198]: Relative is OK for our use as the installer will bundle it all to the same folder. Let me know if changes should be made? Possibly pass the install path from the installer to the client and eventually adjust paths based on that. Open issue?

File: src/FreeScribe.client/UI/SettingsWindow.py [hunk 4]
```diff
@@ -145,6 +146,7 @@ def __init__(self):
             "singleline": False,
             "frmttriminc": False,
             "frmtrmblln": False,
+            "best_of": 6,
```
Comment (yingbull) [inline src/FreeScribe.client/UI/SettingsWindow.py:149]: 2 likely a good number if we are putting one; default to not using this and can be enabled

**Review by yingbull — APPROVED** [review]

**Pull request merged**